iothub/service/src/Common/TrackingHelper.cs

```diff
@@ -220,10 +220,6 @@ public static ErrorCode GetErrorCodeFromAmqpError(Error ex)
             {
                 return ErrorCode.DeviceNotFound;
             }
-            if (ex.Condition.Equals(IotHubAmqpErrorCode.MessageLockLostError))
-            {
-                return ErrorCode.DeviceMessageLockLost;
-            }
```

Comment:
this is dead code for a service application

Reply:
Agreed. However, it is public so just calling it out as a breaking change though in reality it should not be.